packages/core-components/src/lib/unsorted/ui/Tabs/Tabs.svelte

```diff
@@ -40,32 +67,32 @@
 </script>
 
 <section>
-	<nav class="flex gap-x-4 gap-y-1 flex-wrap mb-2">
+	<nav class="my-6 flex flex-wrap gap-x-4 gap-y-1">
 		{#each $tabItems.tabs as tab}
 			<button
+				style:--bgColor={bgColor}
+				style:--borderColor={borderColor}
 				on:click={() => ($tabItems.active = tab.id)}
-				class="px-4 pt-2 border-b-2 border-gray-300 text-sm whitespace-nowrap font-sans"
+				class="mt-2 p-4 rounded-t flex-1 text-sm font-sans whitespace-nowrap transition duration-200 ease-in"
 				class:active={$tabItems.active === tab.id}
 			>
 				{tab.label}
 			</button>
 		{/each}
 	</nav>
-	<div>
+	<div class="p-5 text-base">
 		<slot />
 	</div>
 </section>
 
 <style lang="postcss">
-	nav button {
-		@apply px-4 py-2 border-b-2 border-b-gray-100 hover:border-b-gray-200
-         hover:bg-gray-200
-         active:bg-gray-100
-        transition-colors rounded-t;
-		&.active {
-			@apply border-blue-500
-            hover:border-blue-600 hover:bg-gray-200
-            active:border-blue-700 active:bg-gray-100;
-		}
+	nav button:not(.active) {
+		@apply text-gray-400 hover:bg-gray-200 hover:text-gray-600 active:bg-gray-100;
+	}
+
+	button.active {
+		@apply text-black active:bg-gray-100;
```

Comment:
I believe you can refactor this a little bit to avoid using `@apply`, it has [some performance concerns](https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply).

In the `<script>` section; declare a class string for your `:not(.active)`:

```javascript
const classes = {
    notActive: "text-gray-400 hover:bg-gray-200 hover:text-gray-600 active:bg-gray-100",
    active: "text-black active:bg-gray-100"
}
```

Then in the class string:
```
class="mt-2 p-4 rounded-t flex-1 
            text-sm font-sans 
            whitespace-nowrap
            transition duration-200 ease-in
           { $tabItems.active === tab.id ? classes.active : classes.notActive }
"
```


You can also reference `--bgColor` and `--borderColor` using [tailwind arbitrary values](https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values). You can then include this in your
class strings above

That would look something like this:
```
bg-[--bgColor]
border-b-color-[--borderColor]
border-b-2. (check that this matches what is expected)
border-solid
```

Reply:
One clarification please, 
> bg-[--bgColor]
>border-b-color-[--borderColor]
>border-b-2. (check that this matches what is expected)
>border-solid

Shouldn't the above go in the active property of classes ?
```svelte
const classes = {
    notActive: "text-gray-400 hover:bg-gray-200 hover:text-gray-600 active:bg-gray-100",
    active: "text-black active:bg-gray-100"
}
```